### ISSUE TITLE
Carousel Arrows Missing Focus Styles

### ISSUE THREAD

**Issue by greenyouse** [leandrowd/react-responsive-carousel#546]
**Problem**:
`.control-arrow` buttons are missing focus styles.

**Impact**:
Keyboard only users will not be able to tell where they are in the page without a visual indication for the focused element.

**Recommended Solution**:
Add a visual style for this button so keyboard users know that they're on the arrow button. This could be done by applying the current :hover styles to the arrow elements.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#focusable_element_must_have_focus_styling

**Comment by greenyouse** [comment]
This is looking good. Thanks for adding the change in!
